libs/shared/ui/nuverial/src/lib/components/selector-button-dropdown/selector-button-dropdown.component.ts

```diff
@@ -36,13 +36,17 @@ export class NuverialSelectorButtonDropdownComponent {
    * Input of INuverialSelectOption objects, the displayTextValue key will be displayed in a item inside the menu
    */
   @Input() public menuItems!: INuverialSelectOption[];
+  /**
+   * String that specifies the icon by default it is arrow_drop_down
+   */
+  @Input() public iconName = 'arrow_drop_down';
```

Comment:
Lets make this typesafe please
```suggestion
  @Input() public iconName: 'arrow_drop_down' = 'arrow_drop_down';
```